### PR TITLE
Add documentation for pyretriever, retrieverjl

### PR DESCRIPTION
**Pull request by ShivamNegi**

### CODE REVIEW

**Comment by ShivamNegi** [comment]
Ignore the Julia documentation. All inputs welcome.

**Comment by henrykironde** [comment]
@ShivamNegi, this looks good. Go ahead and create a PR to :python-interface32

File: docs/retrieverjl.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+After installing Data Retriever, ``PyCall`` package needs to be installed.
+
+.. code-block:: julia
+
+    """ Julia Command """
+
+    Pkg.install("PyCall")
+
+The package hasn't been registered in official Julia packages. So each time you want to use Retriever.jl you will have to follow this step.
```
Comment (henrykironde) [inline docs/retrieverjl.rst]: `The package hasn't been registered in official Julia packages. So each time you want to use Retriever.jl you will have to follow this step.` change this to 
To install from Source, download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.

File: docs/retrieverjl.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+==========================
+
+The wrapper module for `Data Retriever <http://data-retriever.org>`_ has been implemented as `Retriever.jl <https://github.com/weecology/retriever.jl.git>`_. All the functions work and feel the same as the python Retriever module. The module has been created using ``PyCall`` hence all the functions are analogous to the functions of retriever python module.
+
+
+Installation
+============
+
+To use Julia package ``retriever.jl``, you first need to `install Retriever <introduction.html#installing-binaries>`_.
+
```
Comment (henrykironde) [inline docs/retrieverjl.rst]: Installation
============

To use Retriever, you first need to `install Retriever  <introduction.html#installing-binaries>`_. a core python package

To  install ``Retriever`` using the Julia package manager

```julia
julia> Pkg.add("Retriever")
```


To install from Source,
download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.

Install  ``PyCall`` package.

.. code-block:: julia

    """ Julia Command """

    Pkg.install("PyCall")


Go to ``retriever.jl/src``. Run Julia.

.. code-block:: julia
    
    """ Julia Command """
    
    include("retriever.jl")


(Note: If you want help in installing Julia you can follow this `tutorial <https://medium.com/@shivamnegi2019/julia-beginners-guide-part-1-a9c369128c78>`_)


.. code-block:: julia

    """ Julia Command """

    Pkg.install("PyCall")

To install from Source, download or checkout the source from the `github page <https://github.com/weecology/retriever.jl.git>`_.

Go to ``retriever.jl/src``. Run Julia.

.. code-block:: julia
    
    """ Julia Command """
    
    include("retriever.jl")


(Note: If you want help in installing Julia you can follow this `tutorial <https://medium.com/@shivamnegi2019/julia-beginners-guide-part-1-a9c369128c78>`_)

**Review by henrykironde — COMMENTED** [review]
what do you think about this

File: docs/pyretriever.rst [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  keywords
+  title
+  urls
+  version
+
+Update Datasets
+===============
+
+If there are no scripts available or you want to update scripts to the latest version.
+``check_for_updates`` function can be used.
```
Comment (ShivamNegi) [inline docs/pyretriever.rst:84]: You missed this review.
>"If there are no scripts available, or you want to update scripts to the latest version,
check_for_updates will download the most recent version of all scripts."

File: docs/pyretriever.rst [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+.. code-block:: python
+
+  >>> rt.check_for_updates()
+
+
+Downloading for the first time the functions will take sometime depending on the internet connection.
```
Comment (ShivamNegi) [inline docs/pyretriever.rst:91]: You missed this review.
> "Downloading recipes for all datasets can take a while depending on the internet connection."

**Pull request closed without merging**